File: C++ Programs/FirstProgram.cpp
#include <iostream>
using namespace std;

int main(int argc, const char *argv[])
{
    string name;
    string music;

    cout << "HELLO WORLD!" << endl;
    cout << "What's your name? ";
    cin >> name;
    cout << "Hello " + name + "!" << endl;
    cout << "What's your favorite kind of music? ";
    cin >> music;
    cout << "Wow that's so cool! I have to go now. Run me again to have another conversation. BYE!" << endl;

    return 0;
}

File: Java/FirstProgram.java
import java.util.Scanner;
public class FirstProgram{
    public static void main (String [] args){
        
        Scanner user = new Scanner (System.in);
        
        System.out.println ("HELLO WORLD");
        System.out.print ("What is your name ");
        String name = user.nextLine();
        System.out.println ("Hello " + name + "!");
        System.out.print ("What's your favorite kind of music? ");
        String music = user.nextLine();
        System.out.println ("Wow that's so cool! I have to go now. Run me again to have another conversation. BYE!");
    }
}